src/marqo/tensor_search/tensor_search.py

```diff
@@ -2146,6 +2295,8 @@ def vectorise_multimodal_combination_field(
                                 f"Could not find image found at `{sub_content}`. \n"
                                 f"Reason: {str(image_repo[sub_content])}"
                             )
+                    elif isinstance(sub_content, str):
```

Comment:
The `NsField.treat_urls_and_pointers_as_images` check in line 2289 is redundant (old code but let's fix it here) as `infer_if_image` covers that

Reply:
Makes sense, removed that check.